pom.xml

```diff
@@ -190,6 +190,28 @@
         <module>xchange-yobit</module>
         <module>xchange-zaif</module>
         <module>xchange-enigma</module>
+        <module>xchange-stream-core</module>
+        <module>xchange-stream-service-pubnub</module>
+        <module>xchange-stream-service-pusher</module>
+        <module>xchange-stream-service-netty</module>
+        <module>xchange-stream-service-wamp</module>
+        <module>xchange-stream-service-core</module>
+        <module>xchange-stream-bitstamp</module>
+        <module>xchange-stream-cexio</module>
+        <module>xchange-stream-okcoin</module>
```

Comment:
should we instead name all these modules like `xchange-okcoin-stream` instead? That way they will end up in our IDEs grouped together better, i.e. `xchange-core` will be next to `xchange-core-stream`.

Reply:
@timmolter We've already renamed using the `xchange-stream` prefix. Can we perhaps aim for a merge first and then rename prior to the next XChange release?  That way I can try and put together the merge sooner rather than later (there are quite a few people screaming for it ;))

Reply:
yeah, we can do the merge first like you suggest. No problem.